src/routes/cart.routes.js

```diff
@@ -1,12 +1,12 @@
 import express from "express";
 import { createCart, viewCart, cleanUpCart } from "../controllers/cart.controller";
 import { validateCartAdd } from "../validations/cart.validator";
-import {verifyToken} from '../middleware/verifyToken';
+import {verifyToken, authorize} from '../middleware/verifyToken';
 
 const router = express.Router();
 
 router.post("/", validateCartAdd, createCart);
 router.get('/view-cart', verifyToken, viewCart);
-router.delete('/clean-up-cart', cleanUpCart);
+router.delete('/clean-up-cart',verifyToken, authorize(['vendor']), cleanUpCart);
```

Comment:
@1MuhireDavid, I think a user who creates a cart is the one supposed to delete it. Not sellers.

Reply:
I think here we can only verify if a user has JWT otherwise let authorize =>`authorize(['buyer'])`

Reply:
> @1MuhireDavid, I think a user who creates a cart is the one supposed to delete it. Not sellers.

Thanks for the comment. Now i decided to make sure that the user who added products on the cart is the one who can delete